api/src/main/java/uk/gov/dhsc/htbhf/claimant/EmailTemplateConfig.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.dhsc.htbhf.claimant;
+
+import uk.gov.dhsc.htbhf.claimant.message.payload.EmailType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Contains all the mappings between email types and template ids. This is built on startup
+ * from configuration only.
+ */
+public class EmailTemplateConfig {
+
+    private final Map<EmailType, String> templateIds = new HashMap<>();
+
```

Comment:
We could consider initialising `EmailTemplateConfig` with a map obtained from yml:
```
public EmailTemplateConfig(@Value("${notify.template-ids") Map<EmailType, String> templateIds) {
    this.templateIds = templateIds;
}
```
And in the yml:
```
notify:
  template-ids:
    NEW_CARD: bbbd8805-b020-41c9-b43f-c0e62318a6d5
```
I've not tried this, but I have a feeling it might work, fingers crossed.

Reply:
That's what I was hoping we might be able to do, didn't know if this worked or not. I'll give it a try.

Reply:
Ok,  got this working by making `EmailMessageConfiguration` a `@ConfigurationProperties`, we can simply add template ids and email types easily then.